os/net/lwip/test/unit/mbox/mbox_shared.c

```diff
@@ -0,0 +1,182 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+ 
+#include <tinyara/config.h>
+#include <assert.h>
+#include <pthread.h>
+#include <queue.h>
+#include <semaphore.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <stress_tool/st_perf.h>
+#include <unistd.h>
+
+#include "mbox_util.h"
+#include "port/port.h"
+#include "sys_arch.h"
+
+#define TCP_QUEUE_SIZE 50
+#define QUEUE_MSG_CNT 100
+static sys_mbox_t g_tcpmbox;
+static sem_t *g_signal_atoc = NULL;
+static sem_t *g_signal_ctoa = NULL;
+static sem_t *g_signal_btoc = NULL;
+static sem_t *g_signal_ctob = NULL;
+
+void mbox_shared_event1(void) {
+  printf("[EVT] let task C know mbox is empty \t%s:%d\n", __FUNCTION__,
```

Comment:
We recommend to use tab instead of space for indentation.

Reply:
Thanks I'll check coding rule to this commit